app/src/main/res/values/preference_keys.xml

```diff
@@ -168,4 +168,5 @@
     <string name="preference_key_payment_methods_gateway_id">paymentMethodsGatewayId</string>
     <string name="preference_key_default_places_location_latlng">placesDefaultLocationLatLng</string>
     <string name="preference_key_recommended_content_survey_shown">recommendedContentSurveyShown</string>
+    <string name="preference_key_user_donor_history">userDonorHistory</string>
```

Comment:
Sorry, I did not make it clear in my previous comment. Would it be possible to also rename it to:
```
<string name="preference_key_donor_history_saved">hasDonorHistorySaved</string>
```

Reply:
done